java/src/jmri/jmrit/operations/trains/TrainCommon.java

```diff
@@ -49,6 +53,8 @@
     protected static final String BLANK_LINE = " ";
     protected static final String HORIZONTAL_LINE_CHAR = "-";
     protected static final String VERTICAL_LINE_CHAR = "|";
+    protected static final String COMMENT_COLOR_START = "<FONT color=\"";
```

Comment:
Please use a `span` element instead of a `font` element here, since the `font` element is not supported in HTML 5, and the JMRI web pages use HTML 5.

Reply:
Not using HTML in this instance, embedding HTML like control characters in plain text.  I couldn't find unicode characters that could tag text color changes.

Reply:
Oh. I was thinking of incorporating that directly into the web-based manifests and conductors.

Reply:
You could, for a route location, in addition to getComment(), there's now getCommentColor(), so that would work just fine.